src/mono/wasm/runtime/startup.ts

```diff
@@ -628,6 +631,56 @@ function downloadResource(request: ResourceRequest): LoadingResource {
     };
 }
 
+export function resolve_module_url(behavior: AssetBehaviours) {
+    let asset: AssetEntry | undefined = config.assets?.find(a => a.behavior == behavior);
+    if (!asset) {
+        switch (behavior) {
+            case "js-module-crypto":
+                asset = {
+                    name: "dotnet-crypto-worker.js",
```

Comment:
Should we rename these files to follow some pattern like `dotnet.crypto.worker.js`, `dotnet.threads.worker.js`?

Reply:
not on this PR, also we I don't know if we could change the name of `dotnet.worker.js` easily until we implement `PThread.init` replacement @lambdageek mentioned.

Reply:
tried `-s PTHREAD_WORKER_FILE=dotnet-threads-worker.js` emcc linker options and got
`PTHREAD_WORKER_FILE is an internal setting and cannot be set from command line`